bindings/netstandard/ElectionGuard/ElectionGuard.Decryption/Decryption/CiphertextDecryptionTally.cs

```diff
@@ -418,9 +429,11 @@ public bool ValidateChallengeResponses()
                 challengeBallots,
                 share,
                 _challenges[guardianId]))
-            {
-                Console.WriteLine("Invalid challenge response for guardian " + guardianId);
+            {
+//                DecryptionState = DecryptionState.PendingAdminComputeProofs;
+
                 return false;
+                throw new ElectionGuardException("Invalid challenge response for guardian " + guardianId);
```

Comment:
Cleaned this up to put the Console.Wrileline back in and to get rid of the throw